book/schedule.md

```diff
@@ -204,10 +204,14 @@ ________________________________________________________
 
 ## Friday July 16
 
-### 9:00 to 10:00 AM: Data Integration
+### 9:00 to 10:00 AM: Project work / open office hours
+
+Join us in the main zoom room if you have any questions!
 
 ### 10:30 to 11:15 AM: Open Source Software
 
+*Lead Instructors: Jessica Scheick, Joachim Meyer and Micah Johnson*
+
 ### 11:30 AM to 12:00 PM: Evaluation Form
 
 ### 12:30 - 3:00 PM Project Presentations

```

Comment:
Would be great to add some links of context here (main zoom room, what is posted on Slack for format of project presentations)

Reply:
good idea - will try to get to this tonight. Wrestling with getting the survey in shape..

Reply:
Should we merge this now so the times are clear and add additional content with a follow up PR?